lib/modules/manager/deno-version/index.ts

```diff
@@ -0,0 +1,36 @@
+import type { Category } from '../../../constants';
+import { NpmDatasource } from '../../datasource/npm';
+import { id, isValid } from '../../versioning/npm';
+
+import type { PackageDependency, PackageFileContent } from '../types';
+
+export const categories: Category[] = ['js'];
+
+export const defaultConfig = {
+  fileMatch: ['(^|/)\\.dvmrc$'],
+  versioning: id,
+};
+
+export const supportedDatasources = [NpmDatasource.id];
+
+export function extractPackageFile(content: string): PackageFileContent | null {
+  if (!content) {
+    return null;
+  }
+
+  if (content.split('\n').length > 2) {
+    return null;
+  }
+
+  const dep: PackageDependency & { currentValue: typeof content } = {
```

Comment:
```suggestion
  const dep: PackageDependency = {
```

the type makes no sense